opentelemetry.html.md.erb

```diff
@@ -51,9 +52,24 @@ otlp:
       PEM_ENCODED_CERTIFICATE
 ```
 
+### <a id='multiple-exporters'></a> Multiple exporters
+
+The OTel Collector supports defining multiple exporters, each with its own configuration. For example, you can define two OTLP gRPC exporters that egress metrics to different destinations.
+
+```
+otlp:
+  endpoint: 203.0.113.10:4317
+otlp/another:
+  endpoint: 203.0.113.11:4317
+```
+
+## <a id='receiver-config'></a> Receiver configuration
+
+Currently, users should not list receivers and include them in the pipeline. By default, receivers are replaced with OTLP receiver that forwarder agent sends to automatically.
+
 ### <a id='grpc-compression'></a> gRPC compression
```

Comment:
Should this be nested under "Receiver configuration" if we're telling operators not to configure receivers? Perhaps it would be more visible in the "Exporter configuration" section, or on its own?

Reply:
Will keep it separate from receivers and exporters